doc: Update helm chart ingress controller setup info

vlasebian opened the pull request:
#### Summary

References issue [#44](https://github.com/TheThingsIndustries/lorawan-stack-k8s/pull/44) in lorawan-stack-k8s repo.

#### Screenshots

<img width="1102" alt="Screenshot 2024-10-23 at 15 48 45" src="https://github.com/user-attachments/assets/3ba3690f-0e36-401e-b577-60419adf12d7">
<img width="684" alt="Screenshot 2024-10-23 at 15 48 54" src="https://github.com/user-attachments/assets/8ccd63ee-66b8-4add-99bb-cd3a18c13a76">
<img width="799" alt="Screenshot 2024-10-23 at 15 49 12" src="https://github.com/user-attachments/assets/1e1ed307-a119-4c7e-b012-708efd75ec65">


#### Changes
<!-- What are the changes made in this pull request? -->

- Update step in prerequisites section of the k8s generic TTS hosting instructions.
- Update affected yaml configuration

#### Notes for Reviewers

- Is there more info need for the ingress controller step or is it enough?

#### Checklist
<!-- Make sure that this pull request is complete. -->

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

KrishnaIyer reviewed (commented):
This looks good. Can you add the Traefik and Nginx examples? 
You can create an `sample-ingress-controllers` doc and add the examples of Traefik and Nginx there.

KrishnaIyer reviewed (commented):
This looks good. Let's get back here once https://github.com/TheThingsIndustries/lorawan-stack-k8s/pull/44 is reviewed and merged in case there are updates from there that need to be adapted here.

vlasebian commented:
@KrishnaIyer I've updated the docs. Let me know if there is anything missing.

I also added the guidelines for setting up support for TTIGPro in the Helm chart.

<img width="1494" alt="Screenshot 2025-01-16 at 15 48 49" src="https://github.com/user-attachments/assets/0095e199-1486-4e32-af2c-62f791b1b2fd" />
<img width="1495" alt="Screenshot 2025-01-16 at 15 49 03" src="https://github.com/user-attachments/assets/5504ffdd-e01b-4b66-9fac-9d0ee70d4053" />
<img width="1500" alt="Screenshot 2025-01-16 at 15 49 10" src="https://github.com/user-attachments/assets/ad375a88-44d5-4650-8db9-47e2ca7e4d36" />

KrishnaIyer reviewed (approved):
Thanks. looks great. Can you please fix the conflicts? You'll need to update the locations since the new PR change
